.github/workflows/discussionbot-workflow.yml

```diff
@@ -0,0 +1,35 @@
+name: Discussion Help Bot
+on:
+  discussion:
+    types:
+    # Use locked as trigger event until this chatbot is ready
+      - created
+    #  - locked
+jobs:
+  run_bot:
+    runs-on: [self-hosted, linux]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: MOOSE_discussion_bot
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Install dependencies
+        run: pip install -r python/MooseDiscussion/requirements.txt
+
+      - name: Run discuss bot
+        run: python python/MooseDiscussion/GitHubBot.py
+
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do you need to explicitly pass this secret? From what I recall github actions use this token by default.

Reply:
I don't think I need to. I have modified this in the new version